external/libecc/Makefile.autosetup

```diff
@@ -138,7 +138,6 @@ CONFLICTS=	-Dsha256_init=_libecc_sha256_init \
 LOCAL_CFLAGS=	-I$(top_srcdir)/external/libecc/include \
 		-I$(top_srcdir)/compat \
 		-I$(top_builddir)/ \
-		-ffreestanding \
 		-fno-builtin \
```

Comment:
We probably don't need `-fno-builtin` either, but I guess it's not actively harmful?  The base version of this is in `^/secure/lib/libpkgecc` if you want to fix that to match, though I guess it doesn't cause problems there.